### ISSUE TITLE
Make Holy Light Holy orders (Silverhand, Argent Dawn, Scarlet etc) more enjoyable

### ISSUE THREAD

**Issue by babosh-gh** [Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth#1508]
Hi!

Is it possible to make the holy light holy orders play like the teutonic knights from the holy order dlc? I think that would make a lot of sense lore-wise
**Describe your suggestion in full detail below:**

**Comment by arithon** [comment]
You mean Holy Fury? Or which DLC?

**Comment by DukeJeans** [comment]
@arithon https://ck2.paradoxwikis.com/Casus_Belli#Northern_Crusade - I think this is what @babosh-gh is referring to

**Comment by arithon** [comment]
So Holy Fury it is...

**Comment by babosh-gh** [comment]
Yes, I was referring to that :D. And to my knowledge, no, there are no great holy wars yet
